src/tsc-errors/classify-tsc-error.test.ts

```diff
@@ -0,0 +1,89 @@
+import { FilePathMatcher } from '../file-path-matcher';
+import classifyTscError from './classify-tsc-error';
+import { TscError, TscErrorMatchFlags } from './types';
+
+describe('classifyTscError', () => {
+  it('should properly classify an ignored error.', () => {
+    const filePath = '/a/file/path.ts';
+    const errorCode = 'TS1234';
+    const tscError: TscError = {
+      filePath: filePath,
+      tscErrorCode: errorCode,
+      rawErrorLines: [],
+    };
+    const filePathMatcher = new FilePathMatcher([filePath]);
+    const ignoredErrorCodes = new Set<string>([errorCode]);
+    const errorClassification = classifyTscError(
+      tscError,
+      ignoredErrorCodes,
+      filePathMatcher,
+    );
+
+    expect(errorClassification).toBe(
+      TscErrorMatchFlags.errorCode | TscErrorMatchFlags.filePath,
+    );
+  });
+
+  it('should properly classify a valid error with error code and file path misses.', () => {
+    const errorFilePath = '/a/file/path.ts';
+    const errorCode = 'TS1234';
+    const tscError: TscError = {
+      filePath: errorFilePath,
+      tscErrorCode: errorCode,
+      rawErrorLines: [],
+    };
+    const filePathToIgnore = '/a/different/file/path.ts';
+    const errorToIgnore = 'TS5678';
+    const filePathMatcher = new FilePathMatcher([filePathToIgnore]);
+    const ignoredErrorCodes = new Set<string>([errorToIgnore]);
+    const errorClassification = classifyTscError(
+      tscError,
+      ignoredErrorCodes,
+      filePathMatcher,
+    );
+
+    expect(errorClassification).toBe(
+      TscErrorMatchFlags.errorCode & TscErrorMatchFlags.filePath,
```

Comment:
Can you just say `0` instead of a bitwise _and_ of 2 values that compute to `0`? IMO just `0` is less confusing and less typo prone

```suggestion
      0,
```